dist/i18n/GUI_Strings_en.txt

```diff
@@ -103,6 +103,7 @@ menu.help.lookUpInDictionary(F1) = Look Up In Dictionary (F1)
 menu.help.netLogoUserManual = NetLogo User Manual
 menu.help.netLogoDictionary = NetLogo Dictionary
 menu.help.netLogoUsersGroup = NetLogo Users Group
+menu.help.netLogoStackOverflow = NetLogo Q & A - StackOverflow
```

Comment:
Can I omit "Stack Overflow" (I mean, "NetLogo Users Group" doesn't mention Yahoo, does it?).